index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Assignment 10</title>
+        <meta charset="utf-8">
+        <link href="style.css" rel="stylesheet">
+    </head>
+    <header>
+        <!--Julie Rizzino
+        index.html
+        INFO 2124
+        Thoendel
+        2/18/20--> 
+    </header>
+    <body>
+        <h1 class=greenText>Assignment 10</h1>
```

Comment:
this should be class="greenText" with the double quotes - and actually - you should set that when someone clicks - not by default.